src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,27 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private String brand;
+    private String model;
+    private String version;
+    private int numberOfDoors;
+    private Engine engine;
 
+    public Car(String brand, String model, String version, int numberOfDoors, Engine engine) {
+        this.brand = brand;
+        this.model = model;
+        this.version = version;
+        this.numberOfDoors = numberOfDoors;
+        this.engine = engine;
+    }
+
+    @Override
+    public Car clone() throws CloneNotSupportedException {
+        brand = brand;
+        model = model;
+        version = version;
+        numberOfDoors = numberOfDoors;
+        engine = engine;
```

Comment:
this makes no sense

Reply:
Originally the task was  - not to use super.clone(). I have changed it now